util/azure/utils.go

```diff
@@ -48,6 +51,14 @@ const (
 	ConnectedClusters           = "Microsoft.Kubernetes/connectedClusters"
 	OperationsEndpointFormatARC = "%s/providers/Microsoft.Kubernetes/operations?api-version=2021-10-01"
 	OperationsEndpointFormatAKS = "%s/providers/Microsoft.ContainerService/operations?api-version=2018-10-31"
+	retrieveServerApiError      = "unable to retrieve the complete list of server APIs"
+	apiserviceUnreachableError  = "the server is currently unable to handle the request"
+)
+
+var (
+	OperationsMapLock   = sync.RWMutex{}
```

Comment:
generally, it's not a good practice to have a globally accessible lock and its protected resource

Reply:
done